loc/en.json

```diff
@@ -444,6 +444,12 @@
     "add_entropy_provide": "Provide entropy via dice rolls",
     "add_entropy_remain": "{gen} bytes of generated entropy. Remaining {rem} bytes will be obtained from the System random number generator.",
     "add_desc": "Creating a new wallet you are generating a single BIP39 seed for BTC and all your Mintlayer tokens. Each asset has a separate chain accessible from your home screen. Keys are derived using BIP84 Hierarchical Deterministic Wallets.",
+    "entrophy_generator": "Entropy generator",
+    "add_desc_entrophy": "In the blank below, please draw anything.",
+    "add_desc_entrophy2": "We are going to use it to randomize your seed.",
+    "add_desc_entrophy3": "It can be a house, a stick man, a dog, or even just a couple of lines or circles.",
```

Comment:
I don't think we want to say this. It defeats the purpose by getting people to draw the same thing

Reply:
How could we exemplify without inducing?

Reply:
Something like "Feel free to draw whatever you please, it could be your favourite animal or shape, an object you can see, or even something more abstract made up of random lines. Let your imagination run wild". 

We've avoided mentioning anything specific which will probably influence a large chunk of people to draw what they've just read. It doesn't have to be exactly the above but something along those lines

Reply:
changed the text